src/dscanner/analysis/config.d

```diff
@@ -195,6 +195,9 @@ struct StaticAnalysisConfig
 	@INI("Check indent of if constraints")
 	string if_constraints_indent = Check.disabled;
 
+    @INI("Check for @trusted applied to a bigger scope than a single function")
+    string trust_too_much = Check.enabled;
```

Comment:
I just noticed you used spaces instead of tabs here

Reply:
Yeah actually there was other spaces indents at the beginning, misleading my editor since it auto adapts to the current style.